docs/services/gracc-corrections.md

```diff
@@ -0,0 +1,62 @@
+Installing GRACC Corrections
+============================
+
+GRACC Corrections are used to modify records during the summarization process.  RAW records are not modified in the correction process.  The correction is applied after summarization and aggregation, but before the record is enriched with data from [Topology](https://topology.opensciencegrid.org/).
+
+The correction is step 3 in the GRACC summary record workflow:
+
+1. Raw record is received.  The raw record is never modified
+2. Summarizer aggregates the raw records
+3. Corrections are applied
+4. Summarized records are enriched by Topology
+5. Summarized and enriched records are uploaded to GRACC
+
+We can currently correct:
+
+* VO Names
+* Project Names
+* OIM_Site (using the `Host_description` field)
+
+Limitations
+-----------
+
+Additional corrections can be written, but some attributes are used to detect duplicate records, and are therefore protected from corrections.  Protected records for summarization are:
```

Comment:
```suggestion
Additional corrections can be written, but some attributes are used to detect duplicate records, and are therefore protected from corrections.  Protected attributes for summarization are:
```

Up to you, but it talks about "attributes" that are "protected" and that those attributes are used to detect duplicate "records", but then it talks about "protected records"